.travis.yml

```diff
@@ -6,17 +6,22 @@ matrix:
   include:
   - python: '2.7'
     env: COVER=1
-  - python: '3.3'
+  # For some reason not working.
+  # - python: '3.3'
```

Comment:
also, what was up with this? Was it transient?

Reply:
not temporary as far as i know. hold on.

Reply:
From Travis log:

```
pip install -r requirements.txt
Obtaining file:///home/travis/build/uber-common/opentracing-python-instrumentation (from -r requirements.txt (line 3))
Exception:
Traceback (most recent call last):
  File "/home/travis/virtualenv/python3.3.6/lib/python3.3/site-packages/pip/basecommand.py", line 215, in main
    status = self.run(options, args)
  File "/home/travis/virtualenv/python3.3.6/lib/python3.3/site-packages/pip/commands/install.py", line 310, in run
    wb.build(autobuilding=True)
  File "/home/travis/virtualenv/python3.3.6/lib/python3.3/site-packages/pip/wheel.py", line 750, in build
    self.requirement_set.prepare_files(self.finder)
  File "/home/travis/virtualenv/python3.3.6/lib/python3.3/site-packages/pip/req/req_set.py", line 370, in prepare_files
    ignore_dependencies=self.ignore_dependencies))
  File "/home/travis/virtualenv/python3.3.6/lib/python3.3/site-packages/pip/req/req_set.py", line 489, in _prepare_file
    abstract_dist.prep_for_dist()
  File "/home/travis/virtualenv/python3.3.6/lib/python3.3/site-packages/pip/req/req_set.py", line 127, in prep_for_dist
    self.req_to_install.run_egg_info()
  File "/home/travis/virtualenv/python3.3.6/lib/python3.3/site-packages/pip/req/req_install.py", line 402, in run_egg_info
    self.setup_py, self.link,
  File "/home/travis/virtualenv/python3.3.6/lib/python3.3/site-packages/pip/req/req_install.py", line 372, in setup_py
    import setuptools  # noqa
  File "/home/travis/virtualenv/python3.3.6/lib/python3.3/site-packages/setuptools/__init__.py", line 12, in <module>
    import setuptools.version
  File "/home/travis/virtualenv/python3.3.6/lib/python3.3/site-packages/setuptools/version.py", line 1, in <module>
    import pkg_resources
  File "/home/travis/virtualenv/python3.3.6/lib/python3.3/site-packages/pkg_resources/__init__.py", line 85, in <module>
    raise RuntimeError("Python 3.4 or later is required")
RuntimeError: Python 3.4 or later is required
make: *** [install-deps] Error 2
```